keyboards/keebio/iris/keymaps/jhelvy/keymap.c

```diff
@@ -0,0 +1,55 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
```

Comment:
It may be worth converting this to an "enum", as it is easier to maintain, long term.